lib/tpm2.c

```diff
@@ -528,6 +525,31 @@ tool_rc tpm2_sequence_complete(ESYS_CONTEXT *esys_context,
     return tool_rc_success;
 }
 
+tool_rc tpm2_event_sequence_complete(
+        ESYS_CONTEXT *ectx,
```

Comment:
every function argument needn't be on the newline, collapse this to character width per coding guideline.